src/routes/parcels/new/+page.ts

```diff
@@ -1,7 +1,10 @@
+import { parcelId } from '../../../stores/sequencing';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async ({ parent }) => {
   const { user } = await parent();
 
+  parcelId.set(null);
```

Comment:
If this is to clear out the `parcelId` set from the `/edit/[id]/+page.ts`, then I don't think it should be the responsibility of other pages to "cleanup" after the other pages. Instead, I suggest removing store logic from both of the `page.ts`, and in just the `edit/[id]/page.svelte`, set the `parcelId` store and clean it up via `onDestroy`